zebra-chain/src/transaction/sighash.rs

```diff
@@ -23,6 +23,8 @@ use crate::primitives::zcash_primitives::sighash;
 
```

Comment:
Most of the code in this file was deleted in PR #2469, so we don't need to make any changes here.

Reply:
Shouldn't this be merged first, so that it's restored when the removal is reverted?

Reply:
PR #2469 has already been merged.

If you'd like, you can add a short summary of these changes to the revert ticket #2468.

Reply:
Done, that's perfect!